docs/en/connector-v2/sink/Clickhouse.md

```diff
@@ -4,7 +4,15 @@
 
 ## Description
 
-Used to write data to Clickhouse. Supports Batch and Streaming mode.
+Used to write data to Clickhouse.
+
+## Key features
+
```

Comment:
Some places have blank lines and others don't

Reply:
done